ch06.adoc

```diff
@@ -30,7 +30,7 @@ string-valued auxiliary coordinate variables. The **`coordinates`** attribute
       other dimensions are axis dimensions. If a character variable has only
       one dimension (the maximum length of the string), it is regarded as a
       string-valued scalar coordinate variable, analogous to a numeric scalar
-      coordinate variable (see <<scalar-coordinate-variables>>)
+      coordinate variable (see <<scalar-coordinate-variables>>).
```

Comment:
The missing full-stop is also missing in the original HTML and PDF versions. Perhaps this fix would be better targetted at v1.7?